testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -97,4 +96,36 @@ public void testTrinoReadingTableCreatedByNativeSpark()
 
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
+
+    @Test(groups = {HIVE_SPARK_BUCKETING, PROFILE_SPECIFIC_TESTS})
+    public void testSparkReadingTableCreatedByNativeTrino()
```

Comment:
- Table -> ParquetTable
  - but rather, we should add coverage for other file formats too, to ensure compatibility with Spark.
- was this test failing before the change?

Reply:
The test didn't exist before the change. I will update to cover Parquet and ORC in both directions Spark -> Trino and Trino -> Spark.

Reply:
> The test didn't exist before the change.

i am aware. Would it fail?

Reply:
It would fail.